process/factory/interceptorscontainer/metaInterceptorsContainerFactory.go

```diff
@@ -62,7 +49,7 @@ func NewInterceptorsContainerFactory(
 	sizeCheckDelta uint32,
 	validityAttester process.ValidityAttester,
 	epochStartTrigger process.EpochStartTriggerHandler,
-) (*interceptorsContainerFactory, error) {
+) (*metaInterceptorsContainerFactory, error) {
```

Comment:
create a method in base that checks common arguments

Reply:
done